### PR TITLE
Complete adventurer craft fix

### PR DESCRIPTION
**Pull request by guyrish**

### CODE REVIEW

File: data/35e/wizards_of_the_coast/supplement/complete_adventurer/cad_skills.lst [hunk 1]
```diff
@@ -1,3 +1,3 @@
 
 
-Craft (Poisonmaking)										KEYSTAT:INT	USEUNTRAINED:NO						TYPE:Intelligence.Craft.Standard.Base.Craft (Poisonmaking)													SOURCEPAGE:p.97
+Craft (Poisonmaking)				KEYSTAT:INT	USEUNTRAINED:NO				TYPE:Intelligence.Craft.Standard.Base	VISIBLE:DISPLAY			SOURCEPAGE:p.97
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/complete_adventurer/cad_skills.lst]: Mate,

I do not understand the intent of this change.

VISIBLE:DISPLAY equates to GUI only. 
Removing the TYPE does not make any sense.

What is this supposed to be fixing?

Comment (guyrish) [inline data/35e/wizards_of_the_coast/supplement/complete_adventurer/cad_skills.lst]: While I don't understand why it's happening the way that it is (as I've tried updating the data a few different ways), the display of the Craft (Poisonmaking) in 6.04 and 6.06.00RC2 both display Craft (Poisonmaking) on a GUI tree node/left peer level to Craft - in fact, Craft (Poisonmaking) appears in the tree as Craft.Craft (Poisonmaking).

As for the VISIBLE tag, that was me mimicking the WotC SRD data file in a vain attempt to get it to display correctly.

Please see the attached images of the "before fix" and "after fix" for a visual representation of the problem from the 6.06.00RC2 release.

craft_before

craft_after

**Comment by BahamutDragon** [comment]
I see. Solution is to change the TYPE to this:

`TYPE:Intelligence.Craft.Standard.Base.Craft (Poisonmaking)`

That will fix the tree issue.

**Comment by guyrish** [comment]
That's what the TYPE string was originally when it displayed incorrectly.

**Comment by BahamutDragon** [comment]
And then go into the pcc file of the set and add this:

`HIDETYPE:SKILL|Craft (Poisonmaking)`

**Comment by guyrish** [comment]
Right. That would remove it. My adjustment puts Poisonmaking into the Craft grouping in the tree.

I agree that my change is strange. However, it fixes the display.

My proposed change, adds Craft (Poisonmaking) to the Craft parent node. Whereas the existing entry, for unknown reasons, forces Craft (Poisonmaking) to be a peer node to Craft, incorrectly making it appear as "Craft.Craft (Poisonmaking)".

craft_after2

**Comment by BahamutDragon** [comment]
I think there is perhaps a misunderstanding. Let me clarify the logic here.

In the PH_SKILLS file, you will see the Craft skills follow this exact TYPE - "Stat.Group.Base.Standard.Skill Name" scheme. 

The PH pcc file hides the extra types like the standard and base and the skill names in the type. What is left to the GUI to display is "Stat.Group", which is the desired output.

To recap: The only adjustment required for this PR to correct the display issue you are seeing is the pcc file for Complete Adventurer. 

Does that now make sense?

**Comment by BahamutDragon** [comment]
`HIDETYPE:SKILL|Strength|Dexterity|Constitution|Intelligence|Wisdom|Charisma|Standard|None|Base|Appraise|Balance|Bluff|Climb|Concentration|Craft (Alchemy)|Craft (Armorsmithing)|Craft (Basketweaving)|Craft (Bookbinding)|Craft (Bowmaking)|Craft (Blacksmithing)|Craft (Calligraphy)|Craft (Carpentry)|Craft (Cobbling)|Craft (Gemcutting)|Craft (Leatherworking)|Craft (Locksmithing)|Craft (Metalworking)|Craft (Painting)|Craft (Pottery)|Craft (Sculpting)|Craft (Shipmaking)|Craft (Stonemasonry)|Craft (Trapmaking)|Craft (Weaponsmithing)|Craft (Weaving)|Decipher Script|Diplomacy|Disable Device|Disguise|Escape Artist|Forgery|Gather Information|Handle Animal|Heal|Hide|Intimidate|Jump|Knowledge (Arcana)|Knowledge (Architecture and Engineering)|Knowledge (Dungeoneering)|Knowledge (Geography)|Knowledge (History)|Knowledge (Local)|Knowledge (Nature)|Knowledge (Nobility and Royalty)|Knowledge (Religion)|Knowledge (The Planes)|Knowledge (Untrained)|Listen|Move Silently|Open Lock|Perform (Act)|Perform (Comedy)|Perform (Dance)|Perform (Keyboard Instruments)|Perform (Oratory)|Perform (Percussion Instruments)|Perform (String Instruments)|Perform (Wind Instruments)|Perform (Sing)|Profession (Apothecary)|Profession (Boater)|Profession (Bookkeeper)|Profession (Brewer)|Profession (Cook)|Profession (Driver)|Profession (Farmer)|Profession (Fisher)|Profession (Guide)|Profession (Herbalist)|Profession (Herder)|Profession (Hunter)|Profession (Innkeeper)|Profession (Lumberjack)|Profession (Miller)|Profession (Miner)|Profession (Porter)|Profession (Rancher)|Profession (Sailor)|Profession (Scribe)|Profession (Siege Engineer)|Profession (Stablehand)|Profession (Tanner)|Profession (Teamster)|Profession (Woodcutter)|Ride|Search|Sense Motive|Sleight of Hand|Speak Language|Spellcraft|Spot|Survival|Swim|Tumble|Use Magic Device|Use Rope|Deception|Athletics|Persuasion|Language|Acrobatics|Perception|Stealth|Animal Empathy|Wilderness Lore|Intuit Direction|Innuedo|Literacy`

**Comment by guyrish** [comment]
I understand now. Thank you for being patient in your explanation.

**Pull request merged**